testing/resources/conda_hooks_repo/.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,5 @@
+-   id: foo
+    name: Foo
+    entry: echo Test
```

Comment:
this doesn't really show that conda works -- it doesn't exercise inside the conda environment